### PR TITLE
add dedicated docker container for some utils

### LINKED ISSUE #57: Add "kopano-utils" image
to make it easy for users to run kopano-backup, kopano-migration-pst, kopano-migration-imap

could be an image that just uses kopano_core as a base then then installs the above listed utilities

### PR DESCRIPTION
**Pull request by fbartels**
fixes https://github.com/zokradonh/kopano-docker/issues/57

Signed-off-by: Felix Bartels <felix@host-consultants.de>

### CODE REVIEW

**Comment by nupplaphil** [comment]
Good idea!

**Comment by fbartels** [comment]
I've added some bits to the readme for this new container. I would say we could merge as soon as @zokradonh has created the build on the docker hub. He is currently out of country, though.

**Comment by fbartels** [comment]
I have merged this manually already, but for whatever reason Github does not seem to realise this. Keeping this open nonetheless as a reminder to create a build in the docker hub for this.

alternatively the travis.yml could be extended to build and publish docker images.

**Review by zokradonh — APPROVED** [review]
hub.docker.com repository created and built.
I like the idea of travis build.

File: utils/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,9 @@
+ARG docker_repo=zokradonh
+FROM ${docker_repo}/kopano_core
+
+RUN apt update && apt-get install --no-install-recommends -y \
```
Comment (zokradonh) [inline utils/Dockerfile:4]: Mixing apt and apt-get here. Better use `apt-get update && apt-get instal...`

**Comment by fbartels** [comment]
manually merged

**Pull request closed without merging**